app/src/main/java/me/iacn/biliroaming/utils/Log.kt

```diff
@@ -17,10 +17,17 @@ object Log {
     fun toast(msg: String, force: Boolean = false, duration: Int = Toast.LENGTH_SHORT) {
         if (!force && !sPrefs.getBoolean("show_info", true)) return
         handler.post {
-            toast?.cancel()
-            toast = Toast.makeText(currentContext, "", duration).apply {
-                setText("哔哩漫游：$msg")
-                show()
+            "com.bilibili.droid.ToastHelper".findClassOrNull(currentContext.classLoader)
+                ?.runCatchingOrNull {
```

Comment:
```suggestion
            ToastHelper?.runCatchingOrNull {
```

然后把 `ToastHelper` 放 Utils 里面 `lazy` 一下。
Utils 里面直接 currentContext.classLoader.loadClass 好了，不需要用 xposed 查